features/Steps/UtilitySteps.php

```diff
@@ -37,4 +39,44 @@ public function iWaitSeconds(int $seconds): void
     {
         sleep($seconds);
     }
+
+    /**
+     * @Given /^I prevent duplicate creation$/
+     */
+    public function iPreventDuplicateCreation(): void
+    {
+        $configFile = file_get_contents('config.php');
+
+        if (str_contains($configFile, "'prevent_duplicate_creation' => true")) {
```

Comment:
Wouldn't it be safer to load the config object into a variable, then check and modify the object, and then serialize it back into the file?
Using `str_contains` and `str_replace` seems a bit more error prone

Reply:
That was what I did first, but Luc advised me to change this to make it more simple.

Reply:
Haha ok never mind then 😉 I didn't look at the resolved comments

Reply:
With the approach of loading the file into a variable an extra conversion was needed when streaming the contents back to the file. See this method https://github.com/cultuurnet/udb3-backend/pull/1578/commits/bc53661bbce982b3f18de08585cf6314045d0ff2#diff-c4ca10529a97c122ff5754d13c511ce2c822c4576c46a797a80c14a1335562a8L78-L88